LayoutTests/imported/w3c/web-platform-tests/css/css-transitions/parsing/transition-valid.html

```diff
@@ -22,6 +22,11 @@
 test_valid_value("transition", "1s -3s cubic-bezier(0, -2, 1, 3) top", "top 1s cubic-bezier(0, -2, 1, 3) -3s");
 test_valid_value("transition", "1s -3s, cubic-bezier(0, -2, 1, 3) top", ["1s -3s, top cubic-bezier(0, -2, 1, 3)", "all 1s ease -3s, top 0s cubic-bezier(0, -2, 1, 3) 0s"]);
 
+test_valid_value("transition", "all", ["all", "all 0s ease 0s"]);
+test_valid_value("transition", "all 1s", ["1s", "all 1s ease 0s"]);
+
+test_valid_value("transition", "initial", ["initial", "initial 0s ease 0s"]);
```

Comment:
I don’t think `initial 0s ease 0s` is a valid result here. That would only happen if the engine incorrectly treated it as a property name. There may be a long serialization that is arguably OK, but I don’t know what it is, I mean `0s ease 0s`, but what property name?

Reply:
That is correct, will only allow `initial` (which works across all three main browser engines).